test/spoom/location_test.rb

```diff
@@ -0,0 +1,100 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+require_relative "../../lib/spoom/location.rb"
+
+require "stringio"
+
+module Spoom
+  module Sorbet
+    class LocationTest < Minitest::Test
+      include Spoom::TestHelper
+
+      def test_from_string
+        location = Location.from_string("foo.rb:1:2-3:4")
+        assert_equal("foo.rb", location.file)
+        assert_equal(1, location.start_line)
+        assert_equal(2, location.start_column)
+        assert_equal(3, location.end_line)
+        assert_equal(4, location.end_column)
+
+        assert_raises(Location::LocationError) do
```

Comment:
(nit) I'd be inclined split these into a separate test so it can be given a meaningful name, e.g. `test_raises_if_location_string_has_missing_components`.

Reply:
Makes sense 👍